docs/analytics_tools/python_libraries.md

```diff
@@ -109,8 +109,8 @@ from siuba import _, filter, count, collect, show_query
 
 # query lastest validation notices, then filter for a single gtfs feed,
```

Comment:
would change comment to align with the query

Reply:
Resolved in [e6f2773](https://github.com/cal-itp/data-infra/pull/2602/commits/e6f2773037116a38d5e1b6783fa7c63729318178)